pkg/cpio/cpio.go

```diff
@@ -84,6 +84,11 @@ func (w Writer) WriteRecord(rec Record) error {
 	return w.rw.WriteRecord(rec)
 }
 
+func (w Writer) WriteReproducibleRecord(rec Record) error {
```

Comment:
I'm not convinced about that. If you ask people to write that function, then you require that users of the package know what makes an array of records reproducible. Over time, this could change. This function should be part of the cpio package. I'm wondering about going back to the model where you can call cpio.Reproducible on a slice of records and have it do what needs to be done.